src/SortingAlgorithms.jl

```diff
@@ -46,6 +47,27 @@ Characteristics:
 """
 const CombSort  = CombSortAlg()
 
+"""
+    BitonicSort
+
+Indicates that a sorting function should use the bitonic mergesort algorithm.
+This algorithm performs a series of pre-determined comparisons, and tends to be very parallelizable.
+The algorithm effectively implements a sorting network based on merging bitonic sequences.
+
+Characteristics:
+ - *not stable* does not preserve the ordering of elements which
+   compare equal (e.g. "a" and "A" in a sort of letters which
+   ignores case).
+ - *in-place* in memory.
+ - *parallelizable* suitable for vectorization with SIMD instructions because
+it performs many independent comparisons.
+
+## References
+- Batcher, K.E., "Sorting networks and their applications", AFIPS '68 (1968), doi: https://doi.org/10.1145/1468075.1468121.
+- "Bitonic Sorter", in "Wikipedia" (Oct 2022). https://en.wikipedia.org/wiki/Bitonic_sorter
```

Comment:
> I'm not really a big fan of bloating the References sections

I agree. I have found the Wikipedia article far more helpful than the paper, and from your comments elsewhere in this PR it seems you may have as well. I think this is a fairly concise and helpful option:

```suggestion
See wikipedia.org/wiki/Bitonic_sorter for more information.
```

Reply:
LGTM!